src/node_lttng_provider.h

```diff
@@ -59,14 +59,17 @@ void NODE_GC_START(v8::GCType type,
   }
   if (flags == v8::GCCallbackFlags::kNoGCCallbackFlags) {
     flagsStr = "kNoGCCallbackFlags";
-  } else if (flags == v8::GCCallbackFlags::kGCCallbackFlagCompacted) {
-    flagsStr = "kGCCallbackFlagCompacted";
+  } else if (flags == v8::GCCallbackFlags::kGCCallbackFlagConstructRetainedObjectInfos) {
+    flagsStr = "kGCCallbackFlagConstructRetainedObjectInfos";
+  } else if (flags == v8::GCCallbackFlags::kGCCallbackFlagForced) {
+    flagsStr = "kGCCallbackFlagForced";
+  } else if (flags == v8::GCCallbackFlags::kGCCallbackFlagSynchronousPhantomCallbackProcessing) {
```

Comment:
That looks a little awkward.  You could add a `using F = v8::GCCallbackFlags` a few lines up so you can shorten the names but I'd probably use an x-macro to generate the clauses.  That lets you drop the duplication of the enum and the string as well.  You can find an example of an x-macro in src/node_v8.cc, grep for HEAP_STATISTICS_PROPERTIES.